All EIPIP Meetings/Meeting 007.md

```diff
@@ -0,0 +1,527 @@
+
+And having the calls, having a cadence of the client developers coming together is something I've seen as important, and cohesion among the clients about what actually is being developed. Without some way of all the clients coming together and regularly talking about it, I don't really see that asynchronously really happening very well.
+
+It's already hard to keep everyone on the same page, and one kind of practical example for hard fork coordination or network upgrades, I actually have zero authority over any client team. I have no way of making any of them do anything. So a really useful thing that has happened on the AllCoreDev calls, and just to be really frank about it, is if we identify something publicly and if I ask what is the status of the implementation of this, and they know every two weeks they have to report on it, it's likely to get done. I could follow up all day and all night, but without that collective accountability, I don't see clients really moving together.
+
+**Tim Beiko**
+So I think there's two bits here, right? One is the implementation parts. I feel like you could get that mostly async, right? So say we have EIP 2315. There is an EthMagicians thread. If Besu hasn't begun our implementation, you can @ us on that thread, and we can comment. I think I agree that around the scheduling and whatnot, that's maybe harder to do, I think, but it feels like we could probably take most decisions async and get more input from the community and have the place where the decision is made be async and then use the calls as the kind of way to resolve these tricky issues that are just hard to resolve async.
+
+And that's already kind of half of what happens, right? Half the conversation happens in the GitHub PR for the EIP, and the only part we're missing there is, like, four clients adding, like, a plus one that, okay, this should be accepted. And that doesn't seem that hard to do async. But I agree that the more fuzzy scheduling is hard.
+
+**Edson Ayllon**
+One comment on that. There has been a need expressed in the survey for clarity on where the EIP process ends and where the hard fork coordination begins. Or at least how those two are separate, clarifying that part.
+
+**Pooja Ranjan**
+I would agree, for the most part, with James as well as Tim. Right now, the process of EIP and the hard fork is not actually separated, but in my opinion, the EthMagicians is a very good forum. We can go ahead and do that in addition to James's idea of collecting information from individual clients, we can also start referring to the point of discussion. And if we make it a preference of having EthMagicians instead of the PR of that, we would be having collective information in one place. And even if it is async, we're also getting async information during the AllCoreDev call.
```

Comment:
I would agree to most of the part, with James as well as Tim. Right now, the process of EIP and the hard fork is not actually separated. In my opinion, the EthMagicians is a very good forum, we can go ahead and do that. In addition to James's idea of collecting information from individual clients, we can also start referring (EthMagician) to the point of discussion. And if we make it a preference of having EthMagicians instead of the PR, we would be having collective information in one place. And even if it is async, we're also getting a synced information during the AllCoreDev call.